### DISCUSSION TITLE
Does lotus clean up storage space when sectors expire?

### DISCUSSION THREAD

**Discussion by Meatball13** [filecoin-project/lotus#5712] [Storage Provider]
Was wondering, as sectors expire I know that the sector will no longer count towards your network power, but does Lotus automatically remove the sectors from storage, or will I need to go in and manually prune them to free the storage space back up?

**Comment by ribasushi** [comment]
cc @magik6k : I assume the answer today is still "no, lotus doesn't auto-clean" ?
